package.json

```
@@ -36,7 +36,8 @@
     "example": "yarn --cwd examples/default",
     "hybrid": "yarn --cwd examples/hybrid",
     "pods": "cd examples/default &&  pod-install --quiet && cd ../hybrid && pod-install --quiet ",
-    "bootstrap": "yarn example && yarn hybrid && yarn && yarn pods"
+    "packSDK": "yarn build && yarn pack --filename InstabugSDK.tgz",
```

We can probably rename this to something like `pack:sdk` rather than `packSDK` to match the naming convention for the rest of the scripts, what do you think?

```suggestion
    "pack:sdk": "yarn build && yarn pack --filename InstabugSDK.tgz",
```